src/ape/managers/converters.py

```diff
@@ -142,25 +142,37 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 
 class TimestampConverter(ConverterAPI):
     """
-    Converts string of ``"%m-%d-%Y %H:%M:%S"`` format to a timestamp.
+    Converts either a string of ``"%m-%d-%Y %H:%M:%S"`` format to a timestamp OR
+    converts a datetime object to a timestamp.
     Must be UTC timezone
     """
 
-    def is_convertible(self, value: str) -> bool:
-        if not isinstance(value, str):
+    def is_convertible(self, value: Union[str, datetime]) -> bool:
+        if not isinstance(value, str) and not isinstance(value, datetime):
             return False
-        if " " not in value or len(value.split(" ")) > 2:
-            return False
-        try:
-            datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
-        except ValueError:
+        if isinstance(value, str):
+            if " " not in value or len(value.split(" ")) > 2:
+                return False
+            else:
+                try:
+                    datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
+                except ValueError:
+                    return False
+        if isinstance(value, datetime) and str(value.tzinfo) != "UTC":
```

Comment:
Is it possible to have a `datetime` object _without_ associated `tzinfo`? I am a little worried users are going to try and use `datetime` objects that do not have timezone information associated with them and their program will fail and they will assume the `TimestampConverter` just doesn't work for `datetime` objects.

Reply:
I'm right there with you on this front. Do we force them to UTC or convert their timestamp to UTC?
If they use it and put in the time they want, and then we convert to UTC, wouldn't that mix up what they want?

Reply:
Are we able to do this:

1. Check if there is not any tz info. If that is the case, it is still convertible and we assume it is UTC.
2. If there _is_ tzinfo and it is NOT UTC, then it is not convertible

So we handle the case where there isn't tz info but still don't convert if there is, if that makes sense.

Reply:
I guess the end user won't see this class because of the abstraction, or I would recommend changing the class name to reflect the dependence on UTC like `UTCTimestampConverter` or something.

Reply:
Warning, then implicitly convert to UTC?

Reply:
We discussed in Discord and don't see a case where timezone should matter. We should just always convert to the non-localized timestamp or assume UTC when there is no timestamp.

A warning might be nice though.... Because maybe the user does not realize they are using localized dates.

Reply:
I removed that, and we just tell the timestamp that this is UTC. The user has to know they have to hand over UTC time